components/profile/Profile.jsx

```diff
@@ -1,11 +1,73 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
+import { Text, View, Alert, SafeAreaView} from 'react-native';
+import { Button, Avatar, List, TextInput} from 'react-native-paper';
 
 export default (props) => {
+  const [name, setName] = React.useState('John Doe');
+  const [state, setState] = React.useState('6x a week');
    return (    
-      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-         <Text>Profile Screen</Text>
-         <Text>Replace these Text tags with your component</Text>
+    <SafeAreaView style={{containier:  {flex: 1}}}>
+      <View style={{flexDirection: 'column', alignItems: "stretch" }}>
```

Comment:
Looks good, one small thing is do you want to have the style be in a styles object at the bottom of the component and then we kind of use that as our stylesheets, it lets us do it like classes and I think its a bit cleaner and easier to read